step-4/build.gradle

```diff
@@ -0,0 +1,80 @@
+buildscript {
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+    classpath "gradle.plugin.io.vertx:vertx-gradle-plugin:0.0.8"
+  }
+}
+
+
+apply plugin: 'java'
+
+group = 'io.vertx'
+version = '1.3.0-SNAPSHOT'
+
+description = """"""
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+tasks.withType(JavaCompile) {
+  options.encoding = 'UTF-8'
+}
+
+
+
+repositories {
+  jcenter()
+}
+
+ext {
+  vertVersion = '3.5.1'
+}
+
+dependencies {
+  compile "io.vertx:vertx-core:$vertVersion"
```

Comment:
Same here, see the Vert.x plugin documentation, it simplifies dependencies management.